x21e8/wallet/sw_wallet.py

```diff
@@ -48,6 +50,11 @@ def get_confidential_liquid_address(self) -> bytes:
     def get_planetmint_pubkey(self) -> bytes:
         return self.public_key
 
+    def get_planetmint_address(self) -> str:
+        cc_key = Ed25519SigningKey(self.private_key, "bytes")
+        encoded_vk = cc_key.get_verifying_key().encode().decode()
```

Comment:
is it really `encoded_vk` if we call `.decode()` last?

Reply:
it's first encoded from bytes to byte-string, then to string. I also don't like how it looks. but the last one is an python intrinsic

Reply:
ok, got it